inc/namespace.php

```diff
@@ -637,28 +672,12 @@ function process_post_ab_test_result( string $test_id, int $post_id ) {
 	// Get existing data for use with queries.
 	$data = get_ab_test_results_for_post( $test_id, $post_id );
 
-	// Bail if test no longer running.
+	// End if test no longer running.
 	if ( ! is_ab_test_running_for_post( $test_id, $post_id ) ) {
-		if ( get_ab_test_end_time_for_post( $test_id, $post_id ) <= milliseconds() ) {
-			// Pause the test.
-			update_is_ab_test_paused_for_post( $test_id, $post_id, true );
-
-			/**
-			 * Dispatch action when test has ended.
-			 *
-			 * @param string $test_id The test ID.
-			 * @param string $post_id The post ID for the test.
-			 * @param array $data The test results so far.
-			 */
-			do_action( 'altis.experiments.test.ended', $test_id, $post_id );
-
-			/**
-			 * Dispatch action when a test of this type has ended.
-			 *
-			 * @param string $post_id The post ID for the test.
-			 * @param array $data The test results so far.
-			 */
-			do_action( "altis.experiments.test.ended.{$test_id}", $post_id );
+		$should_end = get_ab_test_end_time_for_post( $test_id, $post_id ) <= milliseconds();
```

Comment:
Whoa, weird function name

Reply:
They're all like that! You can speak to Joe about it 😛